model-optimizer/mo/utils/ir_engine/ir_engine.py

```diff
@@ -50,7 +57,8 @@ def __load_xml(self):
         self.graph = Graph()
         self.graph.graph['hashes'] = {}
 
-        self.graph.graph['ir_version'] = int(xml_root.attrib['version']) if xml_root.attrib.get('version') is not None else None
+        self.graph.graph['ir_version'] = int(xml_root.attrib['version']) if xml_root.attrib.get(
+            'version') is not None else None
```

Comment:
please return the previous version. Both seem to be the same and the first one is better